lib/bedrock.js

```diff
@@ -610,3 +629,27 @@ function _startWorker(state, script) {
 function _isMessageType(msg, type) {
   return (typeof msg === 'object' && msg.type === type);
 }
+
+function _ensureConfigOverride({config, configOverrideSnapshot}) {
+  const logger = loggers.get('app').child('bedrock/worker');
+  logger.debug('Verifying configuration overrides.');
+  for(const [key, value] of configOverrideSnapshot) {
+    if(lodashGet(config, key) === value) {
```

Comment:
So this doesn't do a deep comparison ... and I'm trying to think if that's a problem or not. I guess if it is, the user would just get more specific with their fields. But there may be cases where that's hard to do, like values that are arrays that you want to be updated.

@mattcollier thoughts? We could do lodash deep equals here if we want to.

Reply:
@dlongley for this pass I think it would be good to be explicit about very specific key/value pairs.  Which which among other things will make it easier to reason about the errors being returned.  Since you're comfortable with the implementation as written, I'm going with it.